tests/environment.nix

```diff
@@ -0,0 +1,12 @@
+{ var, value }:
+
+derivation {
```

Comment:
You can use `mkDerivation` (from `config.nix`) here

Reply:
As you said :ok: